tests/profiler/bug00360.phpt

```diff
@@ -25,8 +25,8 @@ events: Time_(µs) Memory_(bytes)
 
 fl=(1) %sbug00360.php
 fn=(1) func
-2 %d %d
+2 %d %d%r
 
-fl=(2) php:internal
-fn=(2) php::xdebug_get_profiler_filename
-8 %d %d
+fl=\(2\) php:internal
+fn=\(2\) php::xdebug_get_profiler_filename
+8 \d+ \d+|%r
```

Comment:
I don't know why you changed %d to \d, but as I said in the other PR, I'm not OK with this change.

Reply:
because of https://github.com/xdebug/xdebug/pull/604 (this PR is described to be rebased first)

and I find that related PR very clean/robust in sense of tests/flushing